monero-node/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: monero-node
+category: Finance
+name: Monero Node
+version: "0.18.2.2"
+tagline: A simple monero node runner
+description: >-
+  Monero node lets you transact over the secure Monero 
+  blockchain network without the need to trust other parties. 
+releaseNotes: >-
+  This is the most recent version of Monero node.
+developer: 
+website: https://getmonero.prg
+dependencies:
+repo: https://github.com/monero-project/monero
+support: https://github.com/monero-project/monero/issues
+port: 3800
```

Comment:
Is this for the proxy to restricted RPC? This port isn't mentioned anywhere else in the config, just want to be sure it's not an accidental carryover from another config. Would assume it should be `18089`.

Reply:
this is the part I had the biggest WTF moment with, but it is tangled together with 18089 indeed. The magic happens in umbrel-app.yml file

Reply:
Perfect, I trust your knowledge here! Just wanted to double check :smile:

Reply:
the port of the proxy is set to 3800, while in docker-compose.yml you specify where the proxy points towards... tricky, but that's how it works I guess?